app/api/entities/user.rb

```diff
@@ -1,17 +1,23 @@
+# frozen_string_literal: true
+
 module Entities
   class User < Grape::Entity
+    format_with(:iso_timestamp, &:iso8601)
+
     expose :id
     expose :email
     expose :token do |_, options|
       options[:token]
     end
-    expose :last_login do |user, _|
```

Comment:
nope, or better said it didn't make sense in the way it was calculated (the date of the most recent login token) since login tokens are deleted when logging out... we don't really need this currently, so i removed it